src/AppBootstrap.tsx

```diff
@@ -38,10 +40,14 @@ function BlockingBootstrap({ children }: BlockingBootstrapProps): JSX.Element {
   const { bootstrapped: userBootstrapped } = useUser();
   const { bootstrapped: organizationBootstrapped } = useOrganization();
   const { bootstrapped: localizationBootstrapped } = useLocalization();
+  const isAcceleratorRoute = useRouteMatch(APP_PATHS.ACCELERATOR);
```

Comment:
Thoughts on exposing this through a hook? might be good to remove the route match semantics dependency in multiple places.

Reply:
Good call, it seems a bit small for its own hook but this is sprinkled in a lot of places and would be nice to have a location to put more things like this if they come up. Will do that in a follow up PR.

Reply:
Sounds good 👍